lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -1974,29 +1963,42 @@ prop_posAndNegFromCardanoApiValueRoundtrip =
         === v
 
 prop_updateTx
-    :: InAnyRecentEra CardanoApi.Tx
+    :: forall era. era ~ CardanoApi.BabbageEra
```

Comment:
Hi @Anviking 

Just a question:

Does this mean that:
- before this PR, the property was tested for all recent eras; but
- after this PR, the property is only tested for the Babbage era?

Reply:
> after this PR, the property is only tested for the Babbage era?

Yes.

> before this PR, the property was tested for all recent eras

Probably not, but it's not entirely clear.

Reply:
In that case, I think it would be a good idea to put a TODO in the code along the lines of:
```
-- TODO: ADP-XXXX
-- Make this property work in any recent era.
```
(Obviously this requires the existence of a relevant ticket.)

Reply:
Done https://cardanofoundation.atlassian.net/browse/ADP-2997